pennylane/qchem/convert.py

```diff
@@ -226,33 +255,25 @@ def _pennylane_to_openfermion(coeffs, ops, wires=None):
 
     q_op = openfermion.QubitOperator()
     for coeff, op in zip(coeffs, ops):
-        if not isinstance(op, qml.operation.Tensor):
-            op = qml.operation.Tensor(op)
-
-        extra_obsvbs = set(op.name) - {"PauliX", "PauliY", "PauliZ", "Identity"}
-        if extra_obsvbs != set():
+        if isinstance(op, Tensor):
+            try:
+                ps = pauli_sentence(op)
```

Comment:
Does this break for certain `op`s that worked previously with `Tensor`? In that case it might be good to include some info in the error message on how to convert `op` into a valid Pauli word representation manually.

Reply:
Nope, It should work with all those that it used to work with. I just changed the error message to be more descriptive relative to the new types they are working with 👍🏼